install/configuring_inventory_file.adoc

```diff
@@ -313,6 +313,9 @@ of the host. `0.0.0.0/0` is not a valid setting for the IP address.
 *docker* configuration. Block the listed registries. Setting this to `all`
 blocks everything not in the other variables.
 
+|`openshift_docker_ent_reg`
+|An additional registry that is trusted by the container runtime, when `openshift_deployment_type` is set to `openshift-enterprise`. If you set `openshift_docker_ent_reg=''`, then `registry.access.redhat.com` will not be added to the *docker* configuration.
```

Comment:
@bmcelvee, does this note make sense for 3.11?

Reply:
I think it's fine for now. We still don't have a clear timeline for the sunsetting of `registry.access.redhat.com`. If you want, you can replace it with `registry.redhat.io` or maybe "then `registry.access.redhat.com`and `registry.redhat.io` will not..."

Reply:
Thanks @bmcelvee!

@vrutkovs Does it make sense here to say that both`registry.access.redhat.com` and `registry.redhat.io` will not be added?

Reply:
@bergerhoffer sounds good

Reply:
In 3.11, `openshift_docker_ent_reg` default to be `registry.redhat.io` , in 3.10 `openshift_docker_ent_reg` default to be `registry.access.redhat.com`, It maybe better to separate these two?